jobs/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
At some point we can probably want to burn all the migrations back to 0001 so we just have 0001 and 0002 when this goes live, but it can stay for now